composer/loggers/mlflow_logger.py

```diff
@@ -63,12 +73,22 @@ def __init__(
 
         self.run_name = run_name
         self.experiment_name = experiment_name
+        self.model_registry_prefix = model_registry_prefix
+        self.model_registry_uri = model_registry_uri
+        if self.model_registry_uri == 'databricks-uc':
+            if len(self.model_registry_prefix.split('.')) != 2:
+                raise ValueError(f'When registering to Unity Catalog, model_registry_prefix must be in the format ' +
+                                 f'{{catalog_name}}.{{schema_name}}, but got {self.model_registry_prefix}')
+
         self._rank_zero_only = rank_zero_only
         self._last_flush_time = time.time()
         self._flush_interval = flush_interval
         if self._enabled:
             self.tracking_uri = str(tracking_uri or mlflow.get_tracking_uri())
             mlflow.set_tracking_uri(self.tracking_uri)
```

Comment:
@dbczumar @harupy one thing I'm realizing - there's no `MlflowClient` APIs currently for doing things like logging models or achieving functionality like `mlflow.register_model`, etc, which means that library code like this needs to set the global registry/tracking URIs, which affects any user code that runs subsequently and calls MLflow APIs (or the logger would need to remember/restore the original tracking/registry URIs)

Not sure if it's actually possible for additional user code to run after this particular logger is used :) and doesn't seem like a huge issue for MLflow in general, but just something interesting to note

Reply:
yeah, I had to setup the module version of mlflow to match the client version in order for things to go to the right places. i assume that while this logger is active the user would be restricted to the setup of this logger (or if they deviated, things would get weird)